src/Tizen.NUI.Components/Controls/Navigation/DialogPage.cs

```diff
@@ -25,7 +25,7 @@ namespace Tizen.NUI.Components
     /// The DialogPage class is a class which shows a dialog on the page.
     /// DialogPage contains dialog and dimmed scrim behind the dialog.
     /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 9 </since_tizen>
```

Comment:
DialogPage is a Contol. Why don't you use DialogPage's backgroud color instead of seperate scrim View?

Reply:
Scrim catches touch event and DialogPage is popped when user touches Scrim region (dimmed background region).

If we make DialogPage do the above action, the touch point should be checked if it is on DialogPage's children.
(Basically, DialogPage has one child which is "Content" but user can add more views to DialogPage.)

Moreover, if DialogPage catches the touch event, DialogPage should check if a child on the touch point is touchable or not. (e.g. if View.Sensitive is false, then View and its children do not get touch events)

I think that catching the touch event by DialogPage requires us to think more cases to implement Scrim's action.
So I think that let the separate scrim view do the Scrim's action would be easier to maintain.